SU2_CFD/src/solver_direct_heat.cpp

```diff
@@ -314,18 +316,8 @@ void CHeatSolverFVM::LoadRestart(CGeometry **geometry, CSolver ***solver, CConfi
 
   if (flow) {
 
-    if (config->GetKind_Turb_Model() == SA || config->GetKind_Turb_Model() == SA_NEG) {
-      if (nDim == 2) skipVars += 6;
-      if (nDim == 3) skipVars += 8;
-    }
-    else if (config->GetKind_Turb_Model() == SST ) {
-      if (nDim == 2) skipVars += 7;
-      if (nDim == 3) skipVars += 9;
-    }
-    else {
-      if (nDim == 2) skipVars += 5;
```

Comment:
I just noticed in #618 of @rsanfer that he differentiates between SA and SST. Is there any influence here?

Reply:
Hi, thanks for your quick response! :)
The heat solution is written to the solutions files right behind the flow ones since the energy equation became a native part of the incompressible solver as well.
Some of bugfixes in this PR refer to this change, e.g. also the fixes in the preaccumulation.